go.mod

```diff
@@ -1,54 +1,69 @@
 module github.com/ONSdigital/ras-rm-sample/worker
 
-go 1.21
+go 1.23.0
+
+toolchain go1.23.6
```

Comment:
This seems to have been added by `go mod tidy` so will remove it as I don't think its required.